omnigibson/simulator.py

```diff
@@ -99,7 +99,22 @@ def _launch_app():
     if gpu_id is not None:
         config_kwargs["active_gpu"] = gpu_id
         config_kwargs["physics_gpu"] = gpu_id
+
+    # Omni's logging is super annoying and overly verbose, so suppress it by modifying the logging levels
+    if not gm.DEBUG:
+        import sys
+        from numba.core.errors import NumbaPerformanceWarning
+        import warnings
+        sys.argv.append("--/log/level=warning")
+        sys.argv.append("--/log/fileLogLevel=warning")
+        sys.argv.append("--/log/outputStreamLevel=error")
```

Comment:
This seems really brittle, appending arguments to our own call after the fact like this :( if these are arguments that are supposed to be passed into the kit app, can we specify them as part of the config_kwargs passed into SimulationApp in 116? Note that this will be a huge surprise / pain in the ass for any script that still read `sys.argv` _after_ loading the simulator.